### PR TITLE
SMARCB1 deletions in ATRT with current SEG to gene mapping

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
#### Purpose/implementation

In #182, @jharenza suggested we look at _SMARCB1_ deletions in ATRT. This analysis is crossed out on #186 because we were no longer looking to set LRR thresholds, but I was curious and thought it was worth looking at whether these show up given the SEG file to gene symbol mapping introduced in #195.

#### Issue

#186 

#### Directions for reviewers

Rendered notebook is [here](https://jaclyn-taroni.github.io/openpbta-notebook-concept/seg-to-gene-mapping-atrt.nb.html).

#### Results

We don't pick up _SMARCB1_ deletions in ATRT and we don't capture _SMARCB1_ at all. This doesn't mean that the SEG file to gene symbol mapping is wrong, but it is worth being aware of. A possible update to this is to look at the genomic location of _SMARCB1_ in the package(s) we used for the mapping step.

#### Docker and continuous integration

_Check all those that apply or remove this section if it is not applicable._

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
@jashapiro figured out that the `GenomicFeatures::genes` step was filtering out _SMARCB1_ specifically because it was also mapped to some other alternative chromosome. We added filtering to chromosomes 1:22 with https://github.com/jaclyn-taroni/OpenPBTA-analysis/commit/d0850811cc8cd312536d96c3f3d24effb9282b92. I've now run this notebook on the `annotated_cn.tsv` file that included that filtering step and these results look more consistent with what we would expect.

Here's the updated rendered notebook: https://jaclyn-taroni.github.io/openpbta-notebook-concept/seg-to-gene-mapping-atrt-chrom-filter.nb.html

cc @jharenza

**Comment by jaclyn-taroni** [comment]
I forgot to note: these results still are using CNVkit with no notion of ploidy, so the contents of `annotated_cn.tsv` will still probably change but the overall picture looks better.

**Comment by jharenza** [comment]
Yay!! Nice work @jashapiro and @jaclyn-taroni!. One note: we should keep X&Y in if we can, as _ATRX_ is on the X chr. I think, however, the sex chromosomes _may_ have to be handled with caution. When we ran the CN tools, we used the `germline_sex_estimate` to correctly assign sex and therefore ploidy of those chromosomes, and I haven't looked into it, but for males, this _may_ mean X&Y each show a loss of one copy as baseline and therefore. To do that, we can check the tools to assess ploidy for X/Y in males vs females.

**Comment by jaclyn-taroni** [comment]
I've made some updates now that #253 is filed. You can see the rendered notebook here:  https://jaclyn-taroni.github.io/openpbta-notebook-concept/seg-to-gene-mapping-atrt-both-formats.nb.html

To my non-expert eye, this still looks good (cc @jharenza).

**Comment by jharenza** [comment]
Looks good, but are we missing 4 samples from ControlFreeC?

**Comment by jaclyn-taroni** [comment]
For the files I've used here, I chose to filter out `neutral` status so perhaps it could be that step?

https://github.com/jaclyn-taroni/OpenPBTA-analysis/blob/a5f8a6be13db24b3f45ed7e5f6d4b8736e11ed61/analyses/focal-cn-file-preparation/01-prepare-cn-file.R#L144

**Comment by jharenza** [comment]
Ahh ok. I will be interested to see the results of #128 to see if these 4 samples come back with Manta results. If not, I may be concerned that we may miss them. cc: @hongboxie and @fingerfen - something we can revisit once we get consensus calls.

File: analyses/focal-cn-file-preparation/seg-to-gene-mapping-atrt.Rmd [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+From [this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/182#discussion_r340587968) on #182:
+
+> may advise starting with ATRT's and SMARCB1 deletions
+
+Here, I'm looking at the output of `01-prepare-cn-file.R` to see if we recover that information in the segment to gene mapping.
+We're going to get the data from a pull request/branch that is currently open.
+This will get whatever the current files are for that branch.
+
+```{sh}
```
Comment (jashapiro) [inline analyses/focal-cn-file-preparation/seg-to-gene-mapping-atrt.Rmd]: I don't know if `results='hide'` works in shell blocks, but if not, maybe throw a `--quiet` on these?

```suggestion
{sh, results = 'hide'}
```

Comment (jaclyn-taroni) [reply analyses/focal-cn-file-preparation/seg-to-gene-mapping-atrt.Rmd]: Let's try it!

Comment (jaclyn-taroni) [reply analyses/focal-cn-file-preparation/seg-to-gene-mapping-atrt.Rmd]: Doesn't look that way, will do `--quiet`

Comment (jashapiro) [reply analyses/focal-cn-file-preparation/seg-to-gene-mapping-atrt.Rmd]: 😞

**Comment by jaclyn-taroni** [comment]
3 samples are in CNVkit but not ControlFreeC and also ControlFreeC finds one gain which CNVkit marks as a loss (via the choices made over on #253): https://jaclyn-taroni.github.io/openpbta-notebook-concept/seg-to-gene-mapping-atrt-short-comparison.nb.html

**Comment by jaclyn-taroni** [comment]
This is now well covered via the PRs for #244

**Pull request closed without merging**